src/coreclr/inc/stresslog.h

```diff
@@ -225,7 +225,7 @@ class StressLog {
     CRITSEC_COOKIE lock;                    // lock
     uint64_t tickFrequency;         // number of ticks per second
     uint64_t startTimeStamp;        // start time from when tick counter started
-    FILETIME startTime;                     // time the application started
```

Comment:
This field is specified in StressLog cdac, but not used. Is it fine to change it? The bit order of FILETIME should be the same with int64 on LE.

Reply:
I think it is ok